README.md

```diff
@@ -1 +1,125 @@
-# ci
\ No newline at end of file
+# Continuous Integration Server
+
+A continuous integration server written in Java.
+<!-- Add more info here later when we know if we use for example Spring Boot, MongoDB, etc. -->
+
+# What it is
+A continuous integration server that supports:
+* compiling the code
+* running tests
+* notifying users of results
+
+## How to build
+
+**TODO:** Add info on how to build.
+
+```
+git clone git@github.com:sofrel-group14/ci.git
+some other command...
+```
+
+## Tools used
+
+Uses Java (JDK 15) with Maven as build system and JUnit for testing.
+<!-- Add more info here later when we know if we use for example Spring Boot, MongoDB, etc. -->
+
+### Necessary downloads
+
+1. [Download JDK 15](https://www.oracle.com/se/java/technologies/javase-downloads.html)
+2. [Download Maven](https://maven.apache.org/download.cgi) ([installation instructions](https://maven.apache.org/install.html))
+
+# How to contribute
+
+## Code
+
+Code should follow the [Google Java Styleguide](https://google.github.io/styleguide/javaguide.html).
+
+**Each feature has to be associated with at least one test**. See the [Testing](#testing) section for details. This is one of the mandatory grading criterias (P4).
+
+Each public class and method has to be [documented](https://www.monperrus.net/martin/how-to-write-good-API-documentation) using Javadoc. This is one of the mandatory grading criterias (P5).
+
+## Issues
+
+Issues should be categorized using the existing labels (bug, documentation, enhancement, etc.).
+
+~~They should also immediately be linked to the DECIDE project board (specifically the backlog column, i.e. the one chosen by default), so that they are visible there as well.~~  
+**NOTE:** Since we didn't really use the DECIDE-board last assignment, a few of us discussed that maybe it just adds extra unnecessary overhead. Though, if some of you want to use it we can of course do so, in that case we will remove the strikethrough. When we all agree, we will either remove this part altogether or remove the strikethrough.
```

Comment:
I agree I felt as if the decision board was kind of unnecessary for keeping a check on all the different tasks we have to do, I felt like the way we structured issues was good enough.